src/main/resources/python/docs/template_adv.txt

```diff
@@ -8,5 +8,8 @@
 	"read_by_default": "true",
 	"pages": [
 PAGES
-	]
+	],
+	"extra_recipe_mappings": {
+        MAPPINGS
```

Comment:
The indenting on this line is off; see any of the generated files for why.

Reply:
Caught that pycharm decided to use 4 space indents on GenDocs.py, but not here. 

Fixed (in both this and template.txt)